crates/qt-build-utils/src/lib.rs

```diff
@@ -402,6 +402,14 @@ impl QtBuild {
             Err(_) => "lib",
         };
 
+        if let Ok(target) = &target {
+            if target.contains("apple") {
+                // Tell clang link to clang_rt as we build with -nodefaultlibs
+                // otherwise we cannot use helpers from the compiler in Qt
+                println!("cargo:rustc-link-arg=-fapple-link-rtlib");
```

Comment:
can we tell that we are building with clang here? rather than just assuming "apple" == "clang" ?

Reply:
Or do we always use a clang related linker on macOS ? I think yes as it's lld ?

Reply:
Should this be in `qt_build_utils::setup_linker` instead of `QtBuild::cargo_link_libraries` ?

Reply:
done